### PR TITLE
Adding missing references

### PR DESCRIPTION
**Pull request by mcd500**

### CODE REVIEW

File: draft-ietf-teep-protocol.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ informative:
   I-D.ietf-teep-architecture: 
   I-D.lundblade-rats-eat-media-type:
   I-D.ietf-rats-concise-ta-stores:
+  I-D.lundblade-rats-eat-media-type:
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:99]: This line duplicates line 97.  But the correct reference is I-D.ietf-rats-eat-media-type.

File: draft-ietf-teep-protocol.md [hunk 2]
```diff
@@ -96,6 +96,8 @@ informative:
...
+  I-D.wallace-rats-concise-ta-stores:
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:100]: Wrong reference, this was obsoleted by line 98.  Change anything that uses this to use that one.

**Review by dthaler — CHANGES_REQUESTED** [review]

**Comment by dthaler** [comment]
I've now incorporated (I think) the correct fix into PR #313.

**Comment by mcd500** [comment]
> I've now incorporated (I think) the correct fix into PR #313.

Close this PR, move to #313.

**Pull request closed without merging**